### PR TITLE
python, warnings: Multiple code improvements in "fail_over.py" & "fail_back.py"

### PR DESCRIPTION
**Pull request by barpavel**
- Fixing code warnings.
- Local variables' renaming to improve readability, fixing confusing/bad names.
- Different code reorganizations to improve readability.
- Updating "enter vault password" to make failover to be consistent with failback flow.
- Fixing log message that prints a wrong setup (target instead of source):
  "Starting failback DR from setup XXX"
- Adding "TODO"s to be handled later.

Signed-off-by: Pavel Bar <pbar@redhat.com>

### CODE REVIEW

File: files/fail_back.py [hunk 1]
```diff
@@ -37,119 +34,100 @@ def run(self, conf_file, log_file, log_level):
...
-        cmd_fb.append("@" + vault)
-        cmd_fb.append("-e")
-        cmd_fb.append(" dr_target_host=" + target_host
-                      + " dr_source_map=" + source_map
-                      + " dr_report_file=" + report)
-        cmd_fb.append("--vault-password-file")
-        cmd_fb.append("vault_secret.sh")
-        cmd_fb.append("-vvv")
+        dr_failback_tag = "fail_back"
+        extra_vars_failback = " dr_target_host=" + target_host +\
```
Comment (shenitzky) [inline files/fail_back.py:58]: Let's avoid those line breaks

File: files/fail_back.py [hunk 2]
```diff
@@ -37,119 +34,100 @@ def run(self, conf_file, log_file, log_level):
...
+            "--vault-password-file", "vault_secret.sh",
+            "-vvv"
+        ]
 
-        # Setting vault password
-        vault_pass = input(
-            INPUT + PREFIX + "Please enter vault password (In case of "
-            "plain text please press ENTER): " + END)
+        # Setting vault password.
+        vault_pass = input(INPUT
```
Comment (shenitzky) [inline files/fail_back.py:72]: I think that the previous style was more readable.

File: files/fail_back.py [hunk 3]
```diff
@@ -37,119 +34,100 @@ def run(self, conf_file, log_file, log_level):
...
-                 " for oVirt ansible disaster recovery", target_host)
-        print("\n%s%sStarting cleanup process of setup '%s'"
-              " for oVirt ansible disaster recovery%s"
-              % (INFO,
-                  PREFIX,
-                  target_host,
-                  END))
-        log.info("Executing cleanup command: %s", ' '.join(map(str, cmd)))
+
+        info_msg = "Starting cleanup process of setup '{}' for " \
```
Comment (shenitzky) [inline files/fail_back.py:79]: same, please avoid using back-slash

File: files/fail_over.py [hunk 4]
```diff
@@ -22,92 +21,76 @@
...
+            "-t", dr_tag,
+            "-e", "@" + var_file,
+            "-e", "@" + vault,
+            "-e", extra_vars,
+            "--vault-password-file", "vault_secret.sh",
+            "-vvv"
+        ]
+
+        # Setting vault password.
+        vault_pass = input(INPUT
```
Comment (shenitzky) [inline files/fail_over.py:60]: Fit into 2 lines

**Review by shenitzky — CHANGES_REQUESTED** [review]

File: files/fail_over.py (hunk 4 above)
Comment (barpavel) [reply files/fail_over.py:60]: Done, publishing the updated patch now.

File: files/fail_back.py (hunk 2 above)
Comment (barpavel) [reply files/fail_back.py:72]: Publishing the updated patch now.

File: files/fail_back.py (hunk 1 above)
Comment (barpavel) [reply files/fail_back.py:58]: Done, publishing the updated patch now.

File: files/fail_back.py (hunk 3 above)
Comment (barpavel) [reply files/fail_back.py:79]: Done, publishing the updated patch now.

**Review by shenitzky — APPROVED** [review]

**Comment by shenitzky** [comment]
Please add a verification comment so I will be able to merge

**Comment by barpavel** [comment]
Verified today, Sep 24th.

**Pull request merged**